src/System.Net.Http.WinHttpHandler/tests/UnitTests/WinHttpHandlerTest.cs

```diff
@@ -59,7 +59,7 @@ public void Ctor_ExpectedDefaultPropertyValues()
             Assert.Equal(TimeSpan.FromSeconds(30), handler.SendTimeout);
             Assert.Equal(TimeSpan.FromSeconds(30), handler.ReceiveHeadersTimeout);
             Assert.Equal(TimeSpan.FromSeconds(30), handler.ReceiveDataTimeout);
-            Assert.Equal(64 * 1024, handler.MaxResponseHeadersLength);
+            Assert.Equal(64, handler.MaxResponseHeadersLength);
             Assert.Equal(64 * 1024, handler.MaxResponseDrainSize);
```

Comment:
Does DrainSize have a similar issue? Asking just because of the similarity in these test lines.